appendix/troubleshooting/index.rst

```diff
@@ -0,0 +1,126 @@
+   can't find your particular issue described here, feel free to
+   `ask the community`_ for technical assistance.
+
+   .. _ask the community:
+      https://community.zammad.org
+
+Data missing from the Web-UI / Search data missing or incomplete
+================================================================
+
+A commonly reported issue is data missing from the Web-UI.
+This could be Tickets, Articles, Users or anything else
+:doc:`indexed by Elasticsearch </install/elasticsearch/indexed-attributes>`
+and can be caused by missing or incomplete indexes.
+
+If you are experiencing this issue and installed Elasticsearch according to
+:doc:`/install/elasticsearch`, please follow these steps to make sure
+Elasticsearch is working correctly.
+
+Step 1: Verify Elasticsearch is running
+   .. code-block:: sh
+
+      # check elasticsearch status
+      $ systemctl status elasticsearch
+
+   .. note:: 
+      
+      This should output something like the following, make sure it says
+      ``Active: active (running)``:
+
+      .. code-block:: sh
+         :emphasize-lines: 3
+
+         ● elasticsearch.service - Elasticsearch
+            Loaded: loaded (/lib/systemd/system/elasticsearch.service; enabled; vendor preset: enabled)
+            Active: active (running) since Tue 2021-07-20 09:38:21 UTC; 1h 4min ago
+            Docs: https://www.elastic.co
+            Main PID: 1790 (java)
+
+      Otherwise, try starting it and check again:
+
+      .. code-block:: sh
+
+         # restart elasticsearch and check status
+         $ systemctl restart elasticsearch
+         $ systemctl status elasticsearch
+
+      .. warning::
+         
+         | If this fails, your Elasticsearch installation is probably broken.
+         | Try completely purging and reinstalling Elasticsearch according
+           to :doc:`/install/elasticsearch`
+
+
+Step 2: Verify the ingest-attachment plugin is installed correctly
+   .. code-block:: sh
+
+      # list installed elasticsearch plugins
+      $ /usr/share/elasticsearch/bin/elasticsearch-plugin list
+
+   .. note:: 
+      The output should include ``ingest-attachment``.
+
+      Otherwise, try reinstalling the ``ingest-attachment`` plugin and check
+      again:
+
+      .. code-block:: sh
+      
+         $ /usr/share/elasticsearch/bin/elasticsearch-plugin remove ingest-attachment
+         $ /usr/share/elasticsearch/bin/elasticsearch-plugin install ingest-attachment
+
+         $ systemctl restart elasticsearch
+
+         $ /usr/share/elasticsearch/bin/elasticsearch-plugin list
+
+Step 3: Verify Zammad can access Elasticsearch and rebuild the indexes  
+   .. code-block:: sh
+
+      # force zammad to drop and rebuild the elasticsearch indexes
+      $ zammad run rake zammad:searchindex:rebuild
+
+   .. note:: 
+      
+      This should start rebuilding the indexes and output it's progress:
+
+      .. code-block:: sh
+
+         drop indexes...done
```

Comment:
The output changed slightly with https://github.com/zammad/zammad/commit/f678083c3563ebb8538e85268fcd959f9068b6a2. Please run it again with the latest state.